electrolytes/solvation_shell_extract.py

```diff
@@ -0,0 +1,179 @@
+import os
+import logging
+import argparse
+import random
+import numpy as np
+from tqdm import tqdm
+from collections import defaultdict
+import matplotlib.pyplot as plt
+import MDAnalysis as mda
+from solvation_analysis.solute import Solute
+from solvation_analysis._column_names import *
```

Comment:
This is kind bad Python practice. Star imports make it hard to see where things come from and also make it easy to  accidentally shadow other things. Would it be too onerous to ask you to import whatever you need explicitly?